src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingServices/EventsServiceTest.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Services;
+using Altinn.Platform.Events.Tests.Mocks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Altinn.Platform.Events.Tests.TestingServices
+{
+    /// <summary>
+    /// A collection of tests related to <see cref="EventsService"/>.
+    /// </summary>
+    public class EventsServiceTest
```

Comment:
I would like to see some more tests.
1. Ensure ID is set by the service. Maybe just remove it from the event that is to be stored.

Reply:
New test it is :)